platform/ios/src/MGLAPIClient.m

```diff
@@ -87,7 +87,7 @@ - (NSURLRequest *)requestForEvents:(NS_ARRAY_OF(MGLMapboxEventAttributes *) *)ev
 }
 
 - (void)setupBaseURL {
-    NSString *testServerURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMetricsTestServerURL"];
+    NSString *testServerURL = [[NSUserDefaults standardUserDefaults] stringForKey:@"MGLMapboxMetricsTestServerURL"];
```

Comment:
How about using [`-[NSUserDefaults URLForKey:]`](https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSUserDefaults_Class/index.html#//apple_ref/occ/instm/NSUserDefaults/URLForKey:)? You can register the default with an NSURL instance; it'll be stored as a `<data>` value.